cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/ExecutableBuilder.java

```diff
@@ -106,7 +95,6 @@
             asList(ScoreLangConstants.LOOP_KEY, SlangTextualKeys.DO_KEY, SlangTextualKeys.NAVIGATION_KEY);
     private List<String> parallelLoopValidKeywords = asList(SlangTextualKeys.DO_KEY, SlangTextualKeys.FOR_KEY);
 
-    @PostConstruct
```

Comment:
maybe we should leave this as a comment to mark the logic or switch to a builder / constructor approach to call this method in the constructor after all the setters - WDYT? Is it good enough to have the init logic only in spring config class?